### ISSUE TITLE
Suggestion: Only show path between selected nodes

### ISSUE THREAD

**Issue by j-steinbach** [org-roam/org-roam-server#124]
I would like to be able to select two nodes in my graph and then only see the path between those nodes. Everything else should "disappear". 

Example: If I select two nodes _spaghetti_ and _tortellini_ then the following paths should get highlighted

spaghetti -> italian cuisine -> tortellini
spaghetti -> disney movies -> romantic dinner -> tortellini
spaghetti -> trip to italy -> Don Giovanni -> 77 cooking recipes -> tortellini
spaghetti -> wheat -> tortellini

Basically everything that connects node A to node B.

This is helpful to connect two far away ideas. (How could they relate? How do they relate?)

At the moment, it is only possible to highlight the direct neighbours of a node.
